streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImpl.java

```diff
@@ -402,4 +402,9 @@ public TaskType taskType() {
     public Map<TopicPartition, Long> changelogOffsets() {
         return Collections.unmodifiableMap(checkpointFileCache);
     }
+
+    @Override
+    public TopicPartition changelogTopicPartitionFor(final String storeName) {
+        return null;
```

Comment:
Should we better throw an exception as this method should never be called for global state stores?

Reply:
Ack